cloudant-sync-datastore-core/src/test/java/com/cloudant/http/HttpTest.java

```diff
@@ -85,7 +84,8 @@ public InputStream getInputStream() {
     @Test
     public void testWriteToServerOk() throws IOException {
         CouchConfig config = getCouchConfig("httptest"+System.currentTimeMillis());
-        CouchClient client = new CouchClient(config);
+        CouchClient client = new CouchClient(config.getRootUri(), config.getRequestInterceptors()
+                , config.getResponseInterceptors());
```

Comment:
It looks like when doing a line break in other parts of the code, the comma is the last character of the original line and not the first character of the next line.  I prefer that "standard" from a readability POV.  Consider changing this to:

```java
CouchClient client = new CouchClient(config.getRootUri(), config.getRequestInterceptors(),
                config.getResponseInterceptors());
...
```